### ISSUE TITLE
table data serialized and displayed as [object Object]

### ISSUE THREAD

**Issue by 20after4** [HVF/franchise#62]
I'm seeing improperly serialized query results with column values displayed like this:

`[object Object]`

Here is the schema for one of the columns which franchise_client output to the terminal:

```
{   catalog: 'def',
    schema: 'phabricator_maniphest',
    name: 'ownerPHID',
    orgName: 'ownerPHID',
    table: 'maniphest_task',
    orgTable: 'maniphest_task',
    characterSet: 63,
    columnLength: 64,
    columnType: 253,
    flags: 16520,
    decimals: 0 }
```

When I display the results in card view, I get something different:

```json
{"type":"Buffer","data":[80,72,73,68,45,85,83,69,82,45,51,107,99,104,119,107,53,53,100,110]}
```

So it looks like the individual bytes are sent as an array of integers. Is there any way to get franchise to treat those as unicode characters and display them?

**Comment by 20after4** [comment]
I figured out how to make this work. Posted a pull request in case anyone else finds my fix useful, however, it may not be wanted upstream, my apologies if unwanted.

**Comment by 20after4** [comment]
Fixed in 1efa3ccb0c67ad0ba6069f9d5d14cca737143b57
